### PR TITLE
[Pub/sub] Go quickstart

### PR DESCRIPTION
**Pull request by hhunter-ms**
Thank you for helping make the Dapr documentation better!

## Description

- Add Go examples
- Change output from image to code block
- Updated any outdated commands

## Issue reference

The issue this PR will close: #2213

### CODE REVIEW

**Review by msfussell — APPROVED** [review]
LGTM

File: daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md [hunk 1]
```diff
@@ -253,11 +275,35 @@ Notice, as specified in the code above, the publisher pushes a random number to
...
+== APP == Published data: {"orderId":75}
+== APP == Published data: {"orderId":257}
+== APP == Published data: {"orderId":606}
+== APP == Published data: {"orderId":568}
+== APP == Published data: {"orderId":581}
+== APP == Published data: {"orderId":977}
+== APP == Published data: {"orderId":92}
+== APP == Published data: {"orderId":650}
+== APP == Published data: {"orderId":225}
+```
```
Comment (amulyavarote) [inline daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md:290]: There are random numbers here. It should be numbers from 1 to 10.

File: daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md [hunk 2]
```diff
@@ -253,11 +275,35 @@ Notice, as specified in the code above, the publisher pushes a random number to
...
+== APP == Subscriber received: {"orderId":75}
+== APP == Subscriber received: {"orderId":257}
+== APP == Subscriber received: {"orderId":606}
+== APP == Subscriber received: {"orderId":568}
+== APP == Subscriber received: {"orderId":581}
+== APP == Subscriber received: {"orderId":977}
+== APP == Subscriber received: {"orderId":92}
+== APP == Subscriber received: {"orderId":650}
+== APP == Subscriber received: {"orderId":225}
+```
```
Comment (amulyavarote) [inline daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md:307]: There are random numbers here. It should be numbers from 1 to 10.

File: daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md [hunk 3]
```diff
@@ -390,11 +436,33 @@ Notice, as specified in the code above, the publisher pushes a random number to
...
+== APP == Published data: Order { OrderId = 917 }
+== APP == Published data: Order { OrderId = 292 }
+== APP == Published data: Order { OrderId = 722 }
+== APP == Published data: Order { OrderId = 262 }
+== APP == Published data: Order { OrderId = 507 }
+== APP == Published data: Order { OrderId = 73 }
+== APP == Published data: Order { OrderId = 21 }
+== APP == Published data: Order { OrderId = 983 }
+== APP == Published data: Order { OrderId = 886 }
+```
```
Comment (amulyavarote) [inline daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md:451]: There are random numbers here. It should be numbers from 1 to 10.

File: daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md [hunk 4]
```diff
@@ -390,11 +436,33 @@ Notice, as specified in the code above, the publisher pushes a random number to
...
+== APP == Subscriber received: Order { OrderId = 722 }
+== APP == Subscriber received: Order { OrderId = 262 }
+== APP == Subscriber received: Order { OrderId = 507 }
+== APP == Subscriber received: Order { OrderId = 73 }
+== APP == Subscriber received: Order { OrderId = 21 }
+== APP == Subscriber received: Order { OrderId = 983 }
+== APP == Subscriber received: Order { OrderId = 886 }
+```
 
 #### `pubsub.yaml` component file
```
Comment (amulyavarote) [inline daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md:467]: There are random numbers here. It should be numbers from 1 to 10.

File: daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md [hunk 5]
```diff
@@ -537,11 +605,175 @@ Notice, as specified in the code above, the publisher pushes a random number to
 
 Publisher output:
 
-<img src="/images/pubsub-quickstart/pubsub-java-publisher-output.png" width=600 style="padding-bottom:15px;">
+```
+== APP == 2171 [main] INFO com.service.CheckoutServiceApplication - Published data: 0
```
Comment (amulyavarote) [inline daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md]: There is no 0 I believe

File: daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md [hunk 6]
```diff
@@ -390,11 +436,33 @@ Notice, as specified in the code above, the publisher pushes a random number to
...
+== APP == Published data: Order { OrderId = 917 }
+== APP == Published data: Order { OrderId = 292 }
+== APP == Published data: Order { OrderId = 722 }
+== APP == Published data: Order { OrderId = 262 }
+== APP == Published data: Order { OrderId = 507 }
+== APP == Published data: Order { OrderId = 73 }
+== APP == Published data: Order { OrderId = 21 }
+== APP == Published data: Order { OrderId = 983 }
+== APP == Published data: Order { OrderId = 886 }
+```
```
Comment (amulyavarote) [inline daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md:450]: There are random numbers here. It should be numbers from 1 to 10.

File: daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md [hunk 7]
```diff
@@ -390,11 +436,33 @@ Notice, as specified in the code above, the publisher pushes a random number to
...
+== APP == Subscriber received: Order { OrderId = 917 }
+== APP == Subscriber received: Order { OrderId = 292 }
+== APP == Subscriber received: Order { OrderId = 722 }
+== APP == Subscriber received: Order { OrderId = 262 }
+== APP == Subscriber received: Order { OrderId = 507 }
+== APP == Subscriber received: Order { OrderId = 73 }
+== APP == Subscriber received: Order { OrderId = 21 }
+== APP == Subscriber received: Order { OrderId = 983 }
+== APP == Subscriber received: Order { OrderId = 886 }
+```
```
Comment (amulyavarote) [inline daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md:465]: There are random numbers here. It should be numbers from 1 to 10.

File: daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md [hunk 8]
```diff
@@ -253,11 +275,35 @@ Notice, as specified in the code above, the publisher pushes a random number to
...
+== APP == Subscriber received: {"orderId":75}
+== APP == Subscriber received: {"orderId":257}
+== APP == Subscriber received: {"orderId":606}
+== APP == Subscriber received: {"orderId":568}
+== APP == Subscriber received: {"orderId":581}
+== APP == Subscriber received: {"orderId":977}
+== APP == Subscriber received: {"orderId":92}
+== APP == Subscriber received: {"orderId":650}
+== APP == Subscriber received: {"orderId":225}
+```
```
Comment (amulyavarote) [inline daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md:306]: There are random numbers here. It should be numbers from 1 to 10.

**Review by amulyavarote — APPROVED** [review]

**Review by greenie-msft — APPROVED** [review]

**Pull request merged**